### PR TITLE
Display ogv, mp4 and webm videos inside a video tag

### PR DESCRIPTION
**Pull request by cormier**
Add a plugin that displays mp4, ogv and webm videos inside a <video> tag

### CODE REVIEW

File: js/plugins.js [hunk 1]
```diff
@@ -280,6 +280,25 @@ plugins.factory('userPlugins', function() {
     imagePlugin.name = 'image';
 
     /*
+     * mp4 video Preview
+     */
+    var videoPlugin = new Plugin(function(message) {
+
+        var url = message.match(urlRegexp);
```
Comment (lorenzhs) [inline js/plugins.js]: why don't you use `urlPlugin` to easily match multiple videos in one message?

File: js/plugins.js [hunk 2]
```diff
@@ -280,6 +280,25 @@ plugins.factory('userPlugins', function() {
...
+        var content = null;
+
+        if (url) {
+            url = url[0]; /* Actually parse one url per message */
+            if (url.match(/\.(mp4|webm|ogv)$/)) {
+                return '<video width="100%"><source src="'+url+'"></source></video>';
```
Comment (lorenzhs) [inline js/plugins.js]: 100%? Doesn't that make the video huuuuge? I think I'd prefer a Reddit Enhancement Suite-like small default size with click&drag to resize. In any case 100% seems like a bad default to me

Comment (cormier) [inline js/plugins.js]: Maybe that's a little bit big. I don't have a strong opinion on this, so what would you prefer? 75% 50%?

Comment (lorenzhs) [inline js/plugins.js]: Something like youtube (width=560px, height=315px)? I mean, the player has a full-screen button anyway, so we don't need it to be huge.

Comment (cormier) [inline js/plugins.js]: Let's specify only the width to keep the aspect ratio. I would rather keep the "width" a proportion to accomodate different browser sizes though.

**Comment by lorenzhs** [comment]
I don't like the `width=75%` part here, that's way too big imho. Why not use the same as youtube, 560x315 (with a `max-width=100%` for mobile)? The browsers' video player has a full screen button anyway should you want to watch it in huge

**Comment by lorenzhs** [comment]
I changed the width thingy while rebasing it, hope you don't mind

**Pull request merged**